pdns/ueberbackend.hh

```diff
@@ -106,8 +106,8 @@ public:
 
   void getUnfreshSlaveInfos(vector<DomainInfo>* domains);
   void getUpdatedMasters(vector<DomainInfo>& domains, std::unordered_set<DNSName>& catalogs, CatalogHashMap& catalogHashes);
-  bool getDomainInfo(const DNSName& domain, DomainInfo& di, bool getSerial = true);
-  bool createDomain(const DNSName& domain, const DomainInfo::DomainKind kind, const vector<ComboAddress>& masters, const string& account);
+  bool getDomainInfo(const DNSName& domain, DomainInfo& domainInfo, bool getSerial = true);
+  bool createDomain(const DNSName& domain, DomainInfo::DomainKind kind, const vector<ComboAddress>& masters, const string& account);
```

Comment:
`DomainKind` is an enum and is passed by value, `const` is pointless in the declaration and only really useful in the definition. And the caller doesn't need to care whether it's `const` or not because they're passing a copy anyway.

Reply:
I hoped it was something like that, thanks!